dbms/src/main/java/org/polypheny/db/processing/AbstractQueryProcessor.java

```diff
@@ -140,20 +140,59 @@
 @Slf4j
 public abstract class AbstractQueryProcessor implements QueryProcessor {
```

Comment:
Why reordering methods and attributes in this class? Has there been any other changes?